extension/src/fileSystem/index.ts

```diff
@@ -76,6 +77,18 @@ export const findDvcRootPaths = async (cwd: string): Promise<string[]> => {
 
   if (definedAndNonEmpty(subRoots)) {
     dvcRoots.push(...subRoots)
+  } else {
+    const nested = await findFiles(FULLY_NESTED_DVC)
+    if (definedAndNonEmpty(nested)) {
+      dvcRoots.push(
+        ...nested.map(nestedRoot =>
+          parse(nestedRoot)
+            .dir.split(sep)
+            .filter(part => part !== DOT_DVC)
+            .join(sep)
```

Comment:
[Q] Could you use `dirname(dirname(nestedRoot))` here instead or `resolve(nestedRoot,'..', '..')?

The code would look something like this:

```
const nestedConfigs = await findFiles(FULLY_NESTED_DVC)
if (definedAndNonEmpty(nested)) {
  dvcRoots.push(
    ...nestedConfigs.map(config => resolve(config,'..','..')
  )
}
```
I think we also want to call `standardizePath` on the result because of 🪟s.

Reply:
[Q] Do we just want to call this `else` instead of the rest of the function? Will it capture everything? What stops us from ending up with duplicates at the moment?

Reply:
About duplication, `findDvcRootPaths` is only used by  `Setup`'s `findWorkspaceDvcRoots` method and duplication is handled there:

```
const workspaceFolderRoots = await findDvcRootPaths(workspaceFolder)
if (definedAndNonEmpty(workspaceFolderRoots)) {
  dvcRoots = new Set([...dvcRoots, ...workspaceFolderRoots])
  continue
}
```
It is handled there in a loop over workspace folders.

Reply:
I'll rework the whole thing because I think I can remove some more code by going that route.